src/ToDo/Data/TaskData.cs

```diff
@@ -16,6 +16,10 @@ public class TaskData
 	[JsonPropertyName("title")]
 	public string? Title { get; set; }
 
+
+	[JsonPropertyName("displayName")]
+	public string? DisplayName { get; set; }
```

Comment:
The Title property can be removed as this doesn't seem to exist in the new Task definition in the beta api

Reply:
Same before

Reply:
agreed, let's leave it as is